### PR TITLE
Improve development environment

### PR DESCRIPTION
**Pull request by RonanMorgan**
Small fixes in order to make this project work with the pyro-devops development environment 
=> var env update
=> add instruction in the readme
=> add logging in the stdout

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,7 @@
+API_URL=http://localhost
+API_LOGIN=dummy_login
+API_PWD='dummy&P@ssw0rd!'
+
+SENTRY_DSN=
+DEBUG=
+LOGIN=
```
Comment (MateoLostanlen) [inline .env.example:7]: missing blank line

File: Dockerfile [hunk 2]
```diff
@@ -20,3 +20,5 @@ RUN apt update \
 
 # copy project
 COPY app/ /usr/src/app/
+
+CMD python index.py --host 0.0.0.0 --port 8050
```
Comment (MateoLostanlen) [inline Dockerfile:24]: Not sur we need this it's already In the docker compose

Comment (MateoLostanlen) [reply Dockerfile:24]: also missing blank line

**Review by MateoLostanlen — COMMENTED** [review]
Hi Ronan, congratulations on your first PR on the platform. Just a few comments

File: Dockerfile (hunk 2 above)
Comment (RonanMorgan) [reply Dockerfile:24]: I like to have CMD or entrypoint directly in the Dockerfile to be able to use the container without the docker-compose , is it ok ?

=> I thought ruff would fix the blank line, I am gonna do it

Comment (MateoLostanlen) [reply Dockerfile:24]: yes not problem for new just wanted to be sure that it was made on purpose

File: .env.example (hunk 1 above)
Comment (RonanMorgan) [inline .env.example:7]:
```suggestion
LOGIN=

```

Comment (RonanMorgan) [inline .env.example]:
```suggestion
LOGIN=
 
```

File: Dockerfile (hunk 2 above)
Comment (RonanMorgan) [inline Dockerfile:24]:
```suggestion
CMD python index.py --host 0.0.0.0 --port 8050

```

**Pull request merged**